.github/workflows/soaking.yml

```diff
@@ -83,7 +83,7 @@ jobs:
       - name: Get default soaking test configuration
         if: ${{ matrix.language != 'java' && matrix.architecture != 'arm64' }}
         run: |
-          echo SOAKING_TEST_CONFIG="-c 90 -m 70" | tee --append $GITHUB_ENV
+          echo SOAKING_TEST_CONFIG="-c 120 -m 90" | tee --append $GITHUB_ENV
```

Comment:
How have we arrived at the determination that increasing these limits is the correct action and that these are the correct limits?

This seems to invalidate the comment below about the JVM needing more memory.

Reply:
The tests are not failing consistent for the specific layer. based on the previous increased limits in PR's, i was not sure on what basis the limits are raised. at this time I have increased the limits based off of [default config settings](https://github.com/aws-observability/aws-otel-lambda/blob/main/adot/utils/soak/soak.py#L54). I have tested locally with those parameters. but I totally agree there should be some reference on why/how it is increased ?. I Shall add more info in the PR description once I find the relevant information.

Reply:
The principle of alarm threshold could be:
1. CPU usage. Based on the CPU usage of running the sample app without Adot layer. Increase a little bit but not too much;
2. memory consumption. Based on the memory consumption of running the sample app without adot layer, plus the size of layer, then plus RSS. Refer to cgroup memory.max_usage_in_bytes. The RSS value an empirical.

So, before increasing the threshold, please login to test account and manually check the cloudwatch metrics of soaking test. Seems for memory, we don't have to increase threshold from 70 to 90. For CPU, I have no idea why it is over 100 right now, please investigate it is caused by otel dependency update or something else.